wiki/Performance_Troubleshooting/en.md

```diff
@@ -86,3 +86,20 @@ While this may seem like the best option (and indeed can be, if your hardware ca
 - GPUs generally aren’t meant to render 1000+ frames per second, so unexpected behaviour may occur, including lower performance or higher latency than otherwise attainable.
 
 Even so, if heat generation and potentially overworking your hardware is not considered an issue, unlimited may be the option that provides you the most stable frame times.
+
+### Custom
+
+*Note that, if you're using the Cutting Edge release stream, changing the `CustomFrameLimit` to any number above 999, will be reverted to 999; on Stable, you can set the value above 1000 with no issues.*
+*Also note that you cannot switch into this setting with the `F7` shortcut in-game, even if you had done the steps above: you'll need to switch to Custom in the Settings before quitting the game so it'll be available the next time osu! runs. Otherwise, you'll have to change the `FrameSync` manually again.*
```

Comment:
```suggestion
*Notice to Cutting Edge users: changing the `CustomFrameLimit` to any number above 999, will be reverted to 999; on Stable, you can set the value above 1000 with no issues.*

*Caution: you cannot switch into this setting with the `F7` shortcut in-game, even if you do the steps below: you'll need to switch to Custom in the Settings before quitting the game so it'll be available the next time osu! runs. Otherwise, you'll have to change the `FrameSync` manually again.*
```

[hatnotes and notices have specific formatting](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#notice)

also can't have two lines back-to-back because remark and also it's discouraged to have two notices back to back if im not mistaken. But I think it may be fine in this case because the article is so short

Reply:
I've changed it according to the suggestion, with ``changing the `CustomFrameLimit` to`` becomes ``changing the value of `CustomFrameLimit` to``.